screens/chat-screen/home.js

```diff
@@ -312,29 +316,23 @@ export default class Home extends Component {
       this.setState({sleep_tip: data});
       });
     }
-    switch (randAppState) {
+    switch (nextAppState) {
```

Comment:
what was the reason for getting rid of randAppState?

Reply:
My badd!!! this wasn't review ready, should have noted that.
But in answer so we are not returning randomtips.